.github/workflows/pixi_auto_update.yml

```
@@ -8,23 +12,26 @@ on:
   workflow_dispatch:
 
 jobs:
-  auto-update:
+  pixi-update:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - name: Set up pixi
+        uses: prefix-dev/setup-pixi@v0.8.1
         with:
-          ssh-key: ${{ secrets.SSH_PRIVATE_KEY }}
-      - uses: prefix-dev/setup-pixi@v0.8.1
-        with:
-          pixi-version: "latest"
-          cache: false
-      - name: Update pixi lock file
-        run: pixi update
-      - uses: peter-evans/create-pull-request@v6
+          run-install: false
+      - name: Update lockfiles
+        run: |
+          set -o pipefail
+          pixi update --json --no-install | pixi exec pixi-diff-to-markdown >> diff.md
```

```suggestion
          pixi update --json | pixi exec pixi-diff-to-markdown >> diff.md
```

Seems like `--no-install` doesn't work (always?) if pypi dependencies are involved

---

That makes sense, thanks! It works now: https://github.com/Deltares/Ribasim/pull/1708

Shall I add a line about this to https://pixi.sh/latest/advanced/updates_github_actions/?

---

https://github.com/prefix-dev/pixi/issues/1773